www/grpc/proto/blockchain.proto

```diff
@@ -298,8 +298,24 @@ message ConsensusInfo {
   int32 round = 4;
   // List of votes in the consensus instance.
   repeated VoteInfo votes = 5;
+  // The proposal of the consensus instance.
+  Proposal proposal = 6;
 }
 
+// Message containing information about a proposal.
+message Proposal {
+  // The height of the proposal.
+  uint32 height = 1;
+  // The round of the proposal.
+  int32 round = 2;
+  // The block data of the proposal.
+  string block_data = 3;
+  // The signature data of the proposal.
+  string signature_data = 4;
```

Comment:
Should we also include signature `PointG1`?

Reply:
No, Just Signature hex string is enough